storage/storage.hpp

```diff
@@ -40,7 +40,7 @@ namespace bzn
 
         storage_base::result load(const std::string& path) override;
 
-        std::string error_msg(storage_base::result error_id) override;
+        static std::string error_msg(storage_base::result error_id);// override;
```

Comment:
I would not pollute the public interface with this function. I'd put it into the unnamed namespace in the cpp file instead.
There is no need to make this a static and public.

Reply:
removed the error_msg function.